packages/backend-defaults/src/entrypoints/cache/cacheServiceFactory.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CacheManager } from '@backstage/backend-common';
```

Comment:
Should the `CacheManager` be extracted to this package as well?

Reply:
Opening this up for discussion! Happy to expose them from here if we want. Just gotta make sure we're happy to do so and whether it is better to wait for some service extension points to appear instead.